tests/integration/fixtures.py

```diff
@@ -5,10 +5,19 @@
 import numpy as np
 import pandas as pd
 import pytest
+from dask.datasets import timeseries
 from dask.distributed import Client
 from pandas.testing import assert_frame_equal
 
 
+@pytest.fixture()
+def training_df(c):
```

Comment:
Good idea! But maybe a more telling name would be good? Something like "timeseries_df"?

Reply:
sure 👍